lib/plugins/timeago.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+var _ = require('lodash');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['_' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1128614861/issues/source?bid=2452379&fileBranchId=2903028#l3)